dd-sdk-android-ndk/src/main/cpp/utils/datetime-utils.cpp

```diff
@@ -4,15 +4,15 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-#include "datetime.h"
+#include "datetime-utils.h"
 
 #include <cerrno>
 #include <chrono>
 #include <ctime>
 #include <ctime>
 #include <cstdio>
 
-uint64_t time_since_epoch() {
+long long time_since_epoch() {
```

Comment:
Why? 
It is usually better to use an integer type that explicits the number of bits rather than relying on the compiler (which might have different standards)